### ISSUE TITLE
Potential Aria-hidden accessibility issue with reportModal

### ISSUE THREAD

**Issue by kaladay** [TexasDigitalLibrary/Vireo#1276]
I noticed that the `reportModal` does not seem to be changing the `aria-hidden=true` value to `aria-hidden=false` when the modal is displayed.

I believe that when the modal is hidden, `aria-hidden=true` is valid but when the model gets changed from hidden to visible then `aria-hidden=false` should be used to inform the reader/client that that model is now visible.

`aria-hidden` is specific to Accessibility standards from HTML5/wai-aria.
